server/sandbox_run_linux.go

```diff
@@ -31,6 +32,8 @@ import (
 	"k8s.io/kubernetes/pkg/kubelet/types"
 )
 
+const cgroupMemoryPath = "/sys/fs/cgroup/memory"
```

Comment:
nit: This could be better named as `cgroupMemorySubsystemMountPath`

Reply:
Fixed.